src/font/lv_font.h

```diff
@@ -63,6 +69,10 @@ typedef struct {
     lv_font_glyph_format_t format;  /**< Font format of the glyph see lv_font_glyph_format_t */
     uint8_t is_placeholder: 1;      /**< Glyph is missing. But placeholder will still be displayed*/
 
+    /** 0: Get bitmap should return an A8 or ARGB8888 image.
+     * 1: return the bitmap as it is (Maybe A1/2/4 or any proprietary formats). */
+    uint8_t req_raw_bitmap: 1;
```

Comment:
This is not being set to `1` anywhere yet, is it?

Reply:
True, it can be used by some GPUs later.